docs/_docs/SQL/sql-introduction.adoc

```diff
@@ -22,7 +22,7 @@ You can interact with Ignite as you would with any other SQL enabled storage by
 
 Internally, SQL tables have the same data structure as link:data-modeling/data-modeling#key-value-cache-vs-sql-table[key-value caches]. It means that you can change partition distribution of your data and leverage link:data-modeling/affinity-collocation[affinity colocation techniques] for better performance.
 
-Ignite's SQL engine uses H2 Database to parse and optimize queries and generate execution plans.
+Ignite's default SQL engine uses H2 Database to parse and optimize queries and generate execution plans, but also the Apache Calcite-based SQL engine can be enabled to execute queries (experimental feature, see link:SQL/sql-calcite[Calcite-based SQL engine]).
```

Comment:
```suggestion
Ignite's default SQL engine uses H2 Database to parse and optimize queries and generate execution plans, while the Apache Calcite-based SQL engine enables you to execute queries.

WARNING: Apache Calcite-based SQL engine is an experimental feature. For more information, see the link:SQL/sql-calcite[Calcite-based SQL engine, window=_blank] section.
```

Reply:
"Optimize queries and generate execution plans" are phases of "execute queries", these phases can be run using H2 or using Calcite, so Calcite-based engine - it's just an alternative, but not a add-on to do some additional work. I think your suggestion is not quite correct.